msibuilder.ps1

```diff
@@ -11,4 +11,4 @@ Invoke-WebRequest -Uri https://wixtoolset.org/downloads/v3.14.0.4118/wix314.exe
 & "${workdir}/wix314.exe" /q
 
 # Install cmake: used to build openvpn-gui
-& choco.exe install -y cmake
+& choco.exe install -y cmake --installargs 'ADD_CMAKE_TO_PATH=System'
```

Comment:
Looks good. I haven't had issues with this, but if it solves a problem for you then :+1:.

Reply:
Okay, might be worth looking into why it didn't cause any issues for you. Where would I find the openvpn-gui build you mention here?